include/dxc/DXIL/DxilInstructions.h

```diff
@@ -10148,5 +10148,34 @@ struct DxilInst_VectorAccumulate {
   llvm::Value *get_arrayOffset() const { return Instr->getOperand(3); }
   void set_arrayOffset(llvm::Value *val) { Instr->setOperand(3, val); }
 };
+
+/// This instruction n-dimensional vector dot-product
```

Comment:
nit:
```suggestion
/// This instruction computes the n-dimensional vector dot-product
```

Reply:
oh, that explains the formatting for some of the others in the file I looked at for comparison!